src/websockets/client.py

```diff
@@ -364,26 +372,26 @@ class Connect:
 
     def __init__(
         self,
-        uri,
+        uri: str,
         *,
-        create_protocol=None,
-        ping_interval=20,
-        ping_timeout=20,
-        close_timeout=None,
-        max_size=2 ** 20,
-        max_queue=2 ** 5,
-        read_limit=2 ** 16,
-        write_limit=2 ** 16,
-        loop=None,
-        legacy_recv=False,
-        klass=WebSocketClientProtocol,
-        timeout=10,
-        compression="deflate",
-        origin=None,
-        extensions=None,
-        subprotocols=None,
-        extra_headers=None,
-        **kwds
+        create_protocol: Optional[Type[WebSocketClientProtocol]] = None,
+        ping_interval: float = 20,
+        ping_timeout: float = 20,
+        close_timeout: Optional[float] = None,
+        max_size: int = 2 ** 20,
+        max_queue: int = 2 ** 5,
+        read_limit: int = 2 ** 16,
+        write_limit: int = 2 ** 16,
+        loop: Optional[asyncio.AbstractEventLoop] = None,
+        legacy_recv: bool = False,
+        klass: Type[WebSocketClientProtocol] = WebSocketClientProtocol,
+        timeout: float = 10,
+        compression: Optional[str] = "deflate",
+        origin: Optional[str] = None,
+        extensions: Optional[List[ClientExtensionFactory]] = None,
+        subprotocols: Optional[List[str]] = None,
+        extra_headers: Optional[HeadersLike] = None,
+        **kwds: Any
     ):
```

Comment:
Missing `-> None`. There's an option to make mypy complain about these. IIRC it was `--disallow-untyped-defs`.

Reply:
`--disallow-untyped-defs` is implied by `--strict`.

However declaring `-> None` is optional on `__init__` because that's always the expected return type.

Reply:
I realize I included `-> None` for most `__init__` methods. I'm adding the missing ones.